README.md

```diff
@@ -66,6 +66,7 @@ If the VM is no longer needed you can remove it with:
 | cancel_orders  | PUT  |  /orders/\<int:order_id>/cancel  |  Cancel all the items of the Order that have not been shipped yet
 | cancel_item | PUT  | /orders/\<int:order_id>/items/\<int:item_id>/cancel | Cancel a single item in the Order that have not been shipped yet
 | ship_orders  | PUT  |  /orders/\<int:order_id>/ship  |  Ship all the items in an Order
+| deliver_orders  | PUT  |  /orders/\<int:order_id>/deliver  |  Deliver all the items in an Order
```

Comment:
Could you please add it after `ship_item`? Then, it will be in the order of action on the`order` following by that on the`item`, which looks a little bit more organized?

Reply:
That makes sense.

Reply:
sure! Thank you